src/parsing/css-scale-parsing.js

```diff
@@ -28,11 +30,17 @@
     }
     switch (xyOrZ) {
       case 'x':
-        return [new CSSScale(numbers[0], 1), remaining];
+        var result = [new CSSScale(numbers[0], 1), remaining];
+        result[0]._inputType = 'x';
```

Comment:
The same comment I made on your other PR goes for this one too - what do you think?